app/controllers/instructor/sections_controller.rb

```diff
@@ -22,10 +22,11 @@ def update
 	def require_authorizec_for_current_section
 		if current_section.course.user != current_user
 			render text: 'Unauthorized', status: :unauthorized
+		end
```

Comment:
This missing end caused a syntax error when adding a new section.  Without this end, nothing works.